libobs/graphics/effect-parser.c

```diff
@@ -328,7 +513,8 @@ static int ep_parse_pass(struct effect_parser *ep, struct ep_pass *pass)
 
 	if (!cf_token_is(&ep->cfp, "{")) {
 		pass->name = bstrdup_n(ep->cfp.cur_token->str.array,
-		                        ep->cfp.cur_token->str.len);
+			ep->cfp.cur_token->str.len);
```

Comment:
This seems like an unrelated style change to an existing line.  While I _would_ eventually like to bring style consistency to the existing codebase, it's not generally hidden within an existing commit for changing functionality.